### PR TITLE
Add sysinfo attribute to vm xml

### PR DESCRIPTION
**Pull request by meinaLi**
Add sysinfo attribute to vm xml

Signed-off-by: Meina Li <meili@redhat.com>

### CODE REVIEW

**Comment by meinaLi** [comment]
This PR is related with tp-libvirt: https://github.com/autotest/tp-libvirt/pull/3143 which the test result are passed.

**Comment by dzhengfy** [comment]
@meinaLi Please rebase your branch and push it again forcely. The centos 7 failures has been fixed already.

**Review by chloerh — APPROVED** [review]

File: virttest/libvirt_xml/vm_xml.py [hunk 1]
```diff
@@ -3325,3 +3333,24 @@ def __init__(self, virsh_instance=base.virsh):
                                attribute='unit')
         super(VMFeaturesHptXML, self).__init__(virsh_instance=virsh_instance)
         self.xml = '<hpt/>'
+
+
+class VMSysinfoXML(VMXML):
+
+    """
+    sysinfo tag XML class
```
Comment (chunfuwen) [inline virttest/libvirt_xml/vm_xml.py:3341]: Could you paste one simple example how VMSysinfoXML looks like ?

Comment (meinaLi) [reply virttest/libvirt_xml/vm_xml.py]: Updated

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**